[2023-11-02T10:56:42.995+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-04-22T00:00:00+00:00 [queued]>
[2023-11-02T10:56:43.021+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-04-22T00:00:00+00:00 [queued]>
[2023-11-02T10:56:43.024+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T10:56:43.027+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-11-02T10:56:43.029+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T10:56:43.084+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2020-04-22 00:00:00+00:00
[2023-11-02T10:56:43.098+0000] {standard_task_runner.py:55} INFO - Started process 1668 to run task
[2023-11-02T10:56:43.115+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test_dag_tutorial', 'create_postgres_table', 'scheduled__2020-04-22T00:00:00+00:00', '--job-id', '327', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_tutorial.py', '--cfg-path', '/tmp/tmp0uti8vrv']
[2023-11-02T10:56:43.124+0000] {standard_task_runner.py:83} INFO - Job 327: Subtask create_postgres_table
[2023-11-02T10:56:43.310+0000] {task_command.py:376} INFO - Running <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-04-22T00:00:00+00:00 [running]> on host 1dfdc0df4852
[2023-11-02T10:56:43.592+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tetiana
AIRFLOW_CTX_DAG_ID=test_dag_tutorial
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2020-04-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-22T00:00:00+00:00
[2023-11-02T10:56:43.654+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-02T10:56:43.704+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 94, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 288, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 115, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: could not translate host name "host.docker.local" to address: Name or service not known

[2023-11-02T10:56:43.762+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=test_dag_tutorial, task_id=create_postgres_table, execution_date=20200422T000000, start_date=20231102T105642, end_date=20231102T105643
[2023-11-02T10:56:43.871+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 327 for task create_postgres_table (could not translate host name "host.docker.local" to address: Name or service not known
; 1668)
[2023-11-02T10:56:43.951+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-11-02T10:56:44.046+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-02T11:05:48.380+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-04-22T00:00:00+00:00 [queued]>
[2023-11-02T11:05:48.415+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-04-22T00:00:00+00:00 [queued]>
[2023-11-02T11:05:48.416+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T11:05:48.417+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-11-02T11:05:48.418+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T11:05:48.451+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2020-04-22 00:00:00+00:00
[2023-11-02T11:05:48.458+0000] {standard_task_runner.py:55} INFO - Started process 2295 to run task
[2023-11-02T11:05:48.466+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test_dag_tutorial', 'create_postgres_table', 'scheduled__2020-04-22T00:00:00+00:00', '--job-id', '435', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_tutorial.py', '--cfg-path', '/tmp/tmpkez1z1x2']
[2023-11-02T11:05:48.473+0000] {standard_task_runner.py:83} INFO - Job 435: Subtask create_postgres_table
[2023-11-02T11:05:48.589+0000] {task_command.py:376} INFO - Running <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-04-22T00:00:00+00:00 [running]> on host 1dfdc0df4852
[2023-11-02T11:05:48.786+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tetiana
AIRFLOW_CTX_DAG_ID=test_dag_tutorial
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2020-04-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-22T00:00:00+00:00
[2023-11-02T11:05:48.811+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-02T11:05:48.835+0000] {sql.py:315} INFO - Running statement: 
        create table if not exists dag_runs (
        dt date,
        dag_id character varying,
        primary key(dt, dag_id)

        )
        , parameters: None
[2023-11-02T11:05:48.837+0000] {postgres.py:96} INFO - NOTICE:  relation "dag_runs" already exists, skipping

[2023-11-02T11:05:48.867+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=test_dag_tutorial, task_id=create_postgres_table, execution_date=20200422T000000, start_date=20231102T110548, end_date=20231102T110548
[2023-11-02T11:05:48.928+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-11-02T11:05:48.980+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
